libs/core-ui/src/lib/Cohort/CohortBasedComponent.ts

```diff
@@ -45,7 +45,7 @@ export abstract class CohortBasedComponent<
     let selectedCohortName = "";
     let addTemporaryCohort = true;
     if (source === CohortSource.TreeMap || source === CohortSource.HeatMap) {
-      selectedCohortName = "Unsaved";
+      selectedCohortName = "Temporary cohort";
```

Comment:
Actually, this should be localized.  Can you move this into the localization?

Reply:
You do have a good point here... Not sure how I didn't think of that since I have another PR where I did exactly that 🤦🏻‍♂️

Reply:
Done!